src/pages/Getstarted/Getstarted.tsx

```diff
@@ -0,0 +1,37 @@
+import { FC } from 'react'
+import Logo from './logo192.png'
+
+const Getstarted: FC = () => (
+  <div className="flex">
+    <div className="w-1/2 bg-background h-screen flex justify-center items-center">
+      <img src={Logo} className="w-32 h-32" />
+    </div>
+    <div className="w-1/2 bg-white h-screen flex justify-center items-center">
+      <div className="flex flex-col space-y-4 basis-72">
```

Comment:
```suggestion
      <div className="space-y-4 basis-72">
```
dont need flex and flex-col